level/level.go

```diff
@@ -66,6 +75,26 @@ func (l *logger) Log(keyvals ...interface{}) error {
 // Option sets a parameter for the leveled logger.
 type Option func(*logger)
 
+// Allow the provided log level to pass.
+func Allow(v Value) Option {
+	switch v {
+	case debugValue:
+		return AllowDebug()
+
+	case infoValue:
+		return AllowInfo()
+
+	case warnValue:
+		return AllowWarn()
+
+	case errorValue:
+		return AllowError()
+
+	default:
```

Comment:
Stylistically I don't think the blank lines between cases are warranted when each is so simple. I suggest we remove them.